nyxx/lib/src/events/ThreadDeletedEvent.dart

```diff
@@ -0,0 +1,17 @@
+part of nyxx;
+
+/// Fired when a thread is created
+class ThreadDeletedEvent {
+  late final ThreadChannel? thread;
+  late final _ChannelCacheable<TextGuildChannel> parent;

```

Comment:
Better replace that with CacheableTextChannel which implements cacheable and TextChannel interface

Reply:
Can't be found